KK_ButtPlugin/ButtplugController.cs

```diff
@@ -77,6 +116,22 @@ public void Scan()
             StartCoroutine("ScanDevices");
         }
 
+        private string GetPose(int girlIndex = 0)
+        {
+            return flags.nowAnimationInfo.nameAnimation
+                + "." + flags.nowAnimStateName
+                + "." + girlIndex;
+        }
+
+        private float GetPhase(int girlIndex = 0)
+        {
+            if (!animPhases.TryGetValue(GetPose(girlIndex), out float phase))
+            {
+                return 0.0f;  // 

```

Comment:
TryGetValue defaults to zero for float outputs.

Reply:
I'm sure you're starting to realize I don't know C# that well 🙃

Reply:
Take a look at the decompiled Koikatsu assembly. Then look me into the coin and say that again 😆